triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -62,14 +61,15 @@ case class LedgerConfig(
 final case class RunningTrigger(
     triggerInstance: UUID,
     triggerName: Identifier,
-    jwt: Jwt,
+    token: String, // User credentials.
```

Comment:
I think it would be helpful to use a data type (like the `User` type you added to the test code) so we know when we're dealing with credentials and when it's just a username or something else.

Reply:
Also it seems that the username is really a `Party`, so we can the type for that (which I think we were doing before introducing JWTs).